src/app/views/sessions/signin/store/signin.store.ts

```diff
@@ -93,6 +95,15 @@ export class SigninStore extends ComponentStore<SigninState> {
         this.loadFailoverStatus(),
         this.updateService.hardRefreshIfNeeded(),
       ]).pipe(
+        filter(() => {
```

Comment:
I think it's time to extract all login-with-token business into a separate method.